shared/main.tf

```
@@ -3,3 +3,49 @@ module "ecr" {
 
   namespace = local.app_name
 }
+
+module "iam_groups" {
+  source = "../modules/iam/groups"
+}
+
+module "iam_admin_users" {
+  source = "../modules/iam/users"
+
+  usernames = local.iam_admin_emails
+}
+
+module "iam_developer_users" {
+  source = "../modules/iam/users"
+
+  usernames = local.iam_developer_emails
+}
+
+module "iam_bot_users" {
+  source = "../modules/iam/users"
+
+  usernames = local.iam_bot_emails
+}
+
+module "iam_admin_group_membership" {
+  source = "../modules/iam/group_membership"
+
+  name  = "admin_group_membership"
```

```suggestion
  name  = "admin-group-membership"
```

I can't see anything in the Terraform Docs on Compass regarding actual names for variables (only see variable /resource names themselves), however, I think I've seen the hyphen `-` used in this case.

Same for the other ones below if agree :)

---

I don't think so, the names are all snake_case in other places like variables, locals, modules, etc...
So there's no reason to make these outputs become hyphen cases.
What do you think? Can you link me the example that using hyphen as the convention 🙌

---

But we have hyphens here for `groups` 🤔 

https://github.com/nvminhtue/tyson_devops_ic/blob/30d746d13976d941f16c3a3269aa239ce0e394d4/modules/iam/groups/main.tf

```
resource "aws_iam_group" "admin" {
  name = "Admin-group"
}

# trivy:ignore:AVD-AWS-0123
resource "aws_iam_group" "bot" {
  name = "Bot-group"
}

# trivy:ignore:AVD-AWS-0123
resource "aws_iam_group" "developer" {
  name = "Developer-group"
}
```

Which is used on infra the same too:  https://github.com/nimblehq/infrastructure-templates/blob/develop/templates/addons/aws/modules/iam_groups/main.tf

---

ah, for the name, I see
Updated in [3ced894](https://github.com/nvminhtue/tyson_devops_ic/pull/23/commits/3ced894b7cf60e9f1f950d3fdb25556dd6bb025b)